src/workflow/helpers.ts

```diff
@@ -16,6 +17,18 @@ export const needSigningOptionChoice = (prev: any) => {
   }
 };
 
+export const needSurfChoice = (values: Selections) => {
+  if (values.projectType === TemplateProjectType.MOVE) {
+    return null;
+  }
+
+  if (values.template.path === "boilerplate-template") {
```

Comment:
I think this check is enough
```suggestion
  if (values.template.path === FullstackBoilerplateTemplateInfo.value.path) {
```